GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1960,6 +1960,14 @@ private void AddNotes()
 
         private void FindFileInSelectedCommit()
         {
+            var selectedRevisions = RevisionGrid.GetSelectedRevisions();
```

Comment:
nit: in order to emphasize that it is not an enumerator
```suggestion
            IReadOnlyList<GitRevision> selectedRevisions = RevisionGrid.GetSelectedRevisions();
```